core/api/src/services/notifications/index.ts

```diff
@@ -168,7 +192,12 @@ export const NotificationsService = (): INotificationsService => {
     transaction,
   }: NotificatioSendTransactionArgs): Promise<true | NotificationsServiceError> => {
     try {
-      const hasDeviceTokens = recipient.deviceTokens && recipient.deviceTokens.length > 0
+      const settings = await getUserNotificationSettings(recipient.userId)
```

Comment:
this does not make sense, why do we want to persist settings in an external system if we will request them to then send them again to another method of the external system?